lxd-initializer/lxd-initializer-daemonset.yaml

```diff
@@ -47,22 +48,10 @@ spec:
               fieldPath: status.hostIP
         - name: STORAGE_BACKEND
           value: "zfs"
-        - name: STORAGE_SIZE
-          value: "50"
```

Comment:
Lets not remove it, we need way to tweak it with user input if required.

We should handle it.

- if correct user input is provided use it as is.
- If user input is not there, auto-detect resource limit and use it.
- Currently, we have default defined, in case user input is not there. We can improve that default logic with auto-detect